nextflow.config

```
@@ -1,42 +1,26 @@
 // global parameters for workflows
-params{
+params {
   // Workflow metadata
   workflow_url = 'https://github.com/AlexsLemonade/scpca-nf'
 
-  // Docker container images
-  SALMON_CONTAINER = 'quay.io/biocontainers/salmon:1.5.2--h84f40af_0'
-  ALEVINFRY_CONTAINER = 'quay.io/biocontainers/alevin-fry:0.4.2--h7d875b9_0'
-  FASTP_CONTAINER = 'quay.io/biocontainers/fastp:0.23.0--h79da9fb_0'
-  SCPCATOOLS_CONTAINER = 'ghcr.io/alexslemonade/scpca-tools:v0.1.1'
-  CELLRANGER_CONTAINER = '589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:6.1.2'
-  SPACERANGER_CONTAINER = '589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-spaceranger:1.3.1'
-
   // Input data table
   run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
 
   // Output base directory
   outdir = "s3://nextflow-ccdl-results/scpca/processed"
 
-  
-  // Assembly, annotation, and index locations
-  assembly         = 'Homo_sapiens.GRCh38.104'
-  ref_dir          = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104'
-  fasta            = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/fasta/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz'
-  gtf              = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz'
-  index_path       = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_intron.txome'
-  cellranger_index = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/cellranger_index/Homo_sapiens.GRCh38.104_cellranger_full'
-  bulk_index       = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_cdna.txome'
-  mito_file        = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.mitogenes.txt'
-  t2g_3col_path    = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.spliced_intron.tx2gene_3col.tsv'
-  t2g_bulk_path    = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.spliced_cdna.tx2gene.tsv'
-  barcode_dir      = 's3://nextflow-ccdl-data/reference/10X/barcodes' 
- 
   // Processing options
   af_resolution = 'cr-like-em' // alevin-fry quant resolution method: default is cr-like, can also use full, cr-like-em, parsimony, and trivial
   emptydrops_lower = 200 // emptydrops lower bound for cell UMI count
   seed = 2021   // random number seed for filtering (0 means use system seed)
-}
 
+  // Docker container images
+  includeConfig 'config/parameters_containers.config'
+
+  // reference and annotation parameters
+  includeConfig 'config/parameters_reference.config'
+
```

Do we want to simplify these filenames a bit? Not a strong feeling, but `parameters_` seems redundant. 
```suggestion
  includeConfig 'config/containers.config'

  // reference and annotation parameters
  includeConfig 'config/reference_paths.config'

```